mcos/optimizer/markowitz.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+from pypfopt.efficient_frontier import EfficientFrontier
+
+from mcos.optimizer import AbstractOptimizer
+
+
+class MarkowitzOptimizer(AbstractOptimizer):
+    """Optimizer based on the Modern Portfolio Theory pioneered by Harry Markowitz's paper 'Portfolio Selection'""""
+
+    def allocate(self, mu: np.array, cov: np.array) -> np.array:
+        ef = EfficientFrontier(mu, cov)
+        weights = ef.max_sharpe()
```

Comment:
The `EfficientFrontier` class also has a method called `clean_weights` that 

> rounds the weights and clips near-zeros

Think we should be using this?

Reply:
Let's use it. The actual users will appreciate having clean_weights.